Changelog.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.16.0 - 2022-01-07
+
+### Added
+
+-   Log messages from nRF-device-lib-js.
```

Comment:
Maybe a bit more verbose description? E.g. say that there is now an option to toggle nrfdl logging or something similar?